init.js

```diff
@@ -203,7 +203,12 @@ var spawnPoolWorkers = function(){
         } else if (!connection) {
             redisConfig = pcfg.redis;
             connection = redis.createClient(redisConfig.port, redisConfig.host);
-            connection.auth(redisConfig.password);
+            if (redisConfig.password != "") {
+                connection.auth(redisConfig.password);
+                connection.on("error", function (err) {
+                    logger.error("redis", coin, "An error occured while attempting to authenticate redis: " + err);
```

Comment:
The sample error with invalid password:

```
2018-08-03 08:55:22 [redis]     [litecoinz] An error occured while attempting to authenticate redis: ReplyError: ERR invalid password
2018-08-03 08:55:22 [redis]     [litecoinz] An error occured while attempting to authenticate redis: ReplyError: Ready check failed: NOAUTH Authentication required.

```

Reply:
Excellent. This works for me; thanks.

Reply:
You are welcome, thank you for accepting the patch.